### ISSUE TITLE
How to read "memo" / bytes[] / fpt values?

### ISSUE THREAD

**Issue by b-mi** [eXavera/NDbfReader#15]
Hello,

how to read "memo" / bytes[] / fpt values?

Thanks

**Comment by eXavera** [comment]
Hi. By default, there is no support of these column types. But you can use access raw bytes of any column using `GetBytes` and then parse it on your own:
````csharp
using (var table = Table.Open("<file path>"))
{
    var reader = table.OpenReader();

    IColumn yourColumn = table.Columns["<your column>"];

    // have to allocate the buffer on your own
    byte[] rawData = new byte[yourColumn.Size];
    while (reader.Read())
    {
        // read raw bytes of any column
        reader.GetBytes(yourColumn, rawData);

        Console.WriteLine(Convert.ToBase64String(rawData));
    }
}
````

**Comment by b-mi** [comment]
I tried it - this code doesn't work, probably because the dbf file is Foxpro (fpt memo file). The memo column data should always be 10 bytes long, eight of which are zero, the last two have some special values. I spent two days searching for function code to read dbf files. The candidate was NDbfReader, but I could not read the memo fields. I found the solution in another library, where it was obvious that for Foxpro the dbf memo file has a slightly different structure. NDbfReader is fast, and has a well-done option to limit the columns being loaded, plus it has a conversion to a DataTable. Unfortunately, it lacks support for memo fields.. Thanks for the answer.
